contracts/src/instances/corn/CornHyperdrive.sol

```diff
@@ -0,0 +1,99 @@
+///      __  /_/ /__  / / /__  __ \  _ \_  ___/  __  /__  ___/_  /__ | / /  _ \
+///      _  __  / _  /_/ /__  /_/ /  __/  /   / /_/ / _  /   _  / __ |/ //  __/
+///      /_/ /_/  _\__, / _   ___/\___//_/    \__,_/  /_/    /_/  _____/ \___/
+///               /____/   /_/
+///                     XXX          ++          ++          XXX
+///     ############   XXXXX        ++0+        +0++        XXXXX   ###########
+///   ##////////////########       ++00++      ++00++       ########///////////##
+///  ##////////////##########      ++000++    ++000++      ##########///////////##
+///  ##%%%%%%/////      ######     ++0000+    +0000++     ######     /////%%%%%%##
+///    %%%%%%%%&&             ##   ++0000+    +0000++   ##           &&%%%%%%%%%
+///         %&&&                ##  +o000+    +000o+  ##              &&&%
+///                              ## ++00+-    -+00++ ##
+///                               #% ++0+      +0++ %#
+///                               ###-:Oo.++++.oO:-###
+///                                ##: 00++++++00 :##
+///                    #S###########* 0++00+++00++0 *##########S#
+///                  #S               % $ 0+++0 $ %              S#
+///                #S   ----------   %+++++:#:+++++%-----------    S#
+///              #S   ------------- %++++: ### :++++%------------    S#
+///             S    ---------------%++++*\ | /*++++%-------------     S
+///           #S     --------------- %++++ ~W~ ++++%666--o UUUU o-      S#
+///         #S?      ---------------  %+++++~+++++%&&&8 o  \  /  o       ?S#
+///        ?*????**+++;::,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,::;+++**????*?
+///      #?+////////////////////////////////////////////////////////////////+?#
+///    #;;;;;//////////////////////////////////////////////////////////////;;;;;#
+///  S;;;;;;;;;//////////////////////////////////////////////////////////;;;;;;;;;S
+/// /;;;;;;;;;;;///////////////////////////////////////////////////////;;;;;;;;;;;;\
+/// |||OOOOOOOO||OOOOOOOO=========== __  ___        ===========OOOOOOOO||OOOOOOOO|||
+/// |||OOOOOOOO||OOOOOOOO===========|  \[__ |   \  /===========OOOOOOOO||OOOOOOOO|||
+/// |||OOOOOOOO||OOOOOOOO===========|__/[___|___ \/ ===========OOOOOOOO||OOOOOOOO|||
+/// ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+/// |||////////000000000000\\\\\\\\|:::::::::::::::|////////00000000000\\\\\\\\\\|||
+/// SSS\\\\\\\\000000000000////////|:::::0x666:::::|\\\\\\\\00000000000//////////SSS
+/// SSS|||||||||||||||||||||||||||||:::::::::::::::||||||||||||||||||||||||||||||SSS
+/// SSSSSSSS|_______________|______________||_______________|______________|SSSSSSSS
+/// SSSSSSSS                                                                SSSSSSSS
+/// SSSSSSSS                                                                SSSSSSSS
+///
+/// @author DELV
+/// @title CornHyperdrive
+/// @notice A Hyperdrive instance that uses a Corn vault as the yield source.
```

Comment:
Might be nice to have a lil blurb here on what a corn silo is (assuming it's a vault)